client/src/components/ChannelInfo/index.js

```diff
@@ -42,6 +58,13 @@ export default function ChannelInfo(props) {
   const [repoInfo, setRepoInfo] = useState({});
   const [isPrivate, setIsPrivate] = useState(false);
   const [issuesCount, setIssuesCount] = useState(0);
+  const [isLoggedOut, setIsLoggedOut] = useState(false);
```

Comment:
You can fetch this state from the user state in the app which has `isLoggedIn`. 
If the user is not logged in, then fetch the channel members via rc4community backend else fetch directly.